src/test/java/uk/gov/hmcts/reform/sscs/service/SendNotificationServiceTest.java

```diff
@@ -322,28 +322,32 @@ public void sendFallbackLetterNotificationToRep() {
 
     @Test
     public void sendLetterNotificationForAppellant() throws NotificationClientException {
-        classUnderTest.sendLetterNotificationToAddress(buildBaseWrapper(APPELLANT_WITH_ADDRESS, NotificationEventType.CASE_UPDATED), LETTER_NOTIFICATION, APPELLANT_WITH_ADDRESS.getAddress());
+        SubscriptionWithType appellantEmptySubscription = new SubscriptionWithType(EMPTY_SUBSCRIPTION, SubscriptionType.APPELLANT);
+        classUnderTest.sendLetterNotificationToAddress(buildBaseWrapper(APPELLANT_WITH_ADDRESS, NotificationEventType.CASE_UPDATED), LETTER_NOTIFICATION, APPELLANT_WITH_ADDRESS.getAddress(), appellantEmptySubscription.getSubscriptionType());
 
         verify(notificationSender).sendLetter(eq(LETTER_NOTIFICATION.getLetterTemplate()), eq(APPELLANT_WITH_ADDRESS.getAddress()), any(), any());
     }
 
     @Test
     public void sendLetterNotificationForRep() throws NotificationClientException {
```

Comment:
Can we have a test for Rep that does not have a name, but rather has a company/organisation name?

Reply:
Added new test for Rep that does not have a name but org name